src/components/JoinModal/JoinModal.js

```diff
@@ -77,83 +100,105 @@ const JoinModal = () => {
+                  <Button
+                    key={id}
+                    text={title}
+                    isClicked={stackList.includes(id)}
+                    handleClick={() => handleStack(id)}
+                  />
+                ))}
+              </ButtonBox>
+              <GitHub>
+                <Title>* Git 주소</Title>
+                <GitInput
+                  name="git"
+                  placeholder="깃헙 링크 넣어주삼"
                   onChange={handleTextChange}
-                  value="frontend"
                 />
-                <Label htmlFor="checkBox">front end</Label>
-              </CheckBox>
-              <CheckBox>
-                <ChoiceBtn
-                  type="radio"
-                  id="choiceBox"
-                  name="position"
-                  onChange={handleTextChange}
-                  value="backend"
-                />
-                <Label htmlFor="choiceBox">back end</Label>
-              </CheckBox>
-            </CheckBoxWrap>
-            <ButtonBox>
-              <Title>* 기술 스택</Title>
-              {STACK_LIST.map(stack => (
-                <Button
-                  key={stack.id}
-                  text={stack.name}
-                  isClicked={stackList.includes(stack.name)}
-                  handleClick={() => handleStack(stack.name)}
-                />
-              ))}
-            </ButtonBox>
-            <GitHub>
-              <Title>* Git 주소</Title>
-              <GitInput
-                name="git"
-                placeholder="깃헙 링크 넣어주삼"
-                onChange={handleTextChange}
-              ></GitInput>
-            </GitHub>
-            <OpenCheckPortfolio>
-              <CheckBox>
-                <ChoiceBtn
-                  type="checkbox"
-                  id="choicebox"
-                  name="choiceBox"
-                  onChange={choicePortfolio}
-                ></ChoiceBtn>
-                <Label htmlFor="choicebox">내 포트폴리오 공개여부</Label>
-              </CheckBox>
-            </OpenCheckPortfolio>
-            <JoinBtn onClick={handleBtn}>apply</JoinBtn>
-          </ModalForm>
-        </Modal>
-      )}
-    </>
+              </GitHub>
+              <OpenCheckPortfolio>
+                <CheckBox>
+                  <Label htmlFor="choicebox">
+                    <ChoiceBtn
+                      type="checkbox"
+                      id="choicebox"
+                      name="choiceBox"
+                      onChange={choicePortfolio}
+                    />
+                    내 포트폴리오 공개여부
+                  </Label>
+                </CheckBox>
+              </OpenCheckPortfolio>
+              <JoinBtn onClick={handleBtn}>apply</JoinBtn>
+            </ModalForm>
+          </Modal>
+        )}
+      </>
+    )
   );
 };
 const GoToJoin = styled.button`
-  background-color: red;
+  width: 100%;
+  padding: 16px;
+  margin-top: 16px;
+  border-radius: 3px;
+  font-size: 12px;
+  border: none;
+
+  :hover {
```

Comment:
의도적으로 하신 부분이 아니라면, 부모 가상 선택자(`&`)를 붙여서 자식 요소가 아니라 button 자체에 hover 효과를 주는 게 더 좋을 것 같습니다.
```suggestion
  &:hover {
```